src/main/java/org/broadinstitute/consent/http/service/ElectionService.java

```diff
@@ -184,11 +188,20 @@ public Election updateElectionById(Election rec, Integer electionId) {
         return electionDAO.findElectionWithFinalVoteById(electionId);
     }
 
-    public Election submitFinalAccessVoteDataRequestElection(Integer electionId) throws Exception {
+    public Election submitFinalAccessVoteDataRequestElection(Integer electionId, String darId) throws Exception {
         Election election = electionDAO.findElectionWithFinalVoteById(electionId);
```

Comment:
The election has a reference id that is the DAR id, so you don't need to pass that in as an argument.